### PR TITLE
Specify that mitigating privacy risks is an open problem.

### PR DESCRIPTION
**Pull request by cbartle891**
Related issue and discussion: https://github.com/tlswg/external-psk-design-team/issues/36#issuecomment-675796072

### CODE REVIEW

File: draft-ietf-tls-external-psk-guidance.md [hunk 1]
```diff
@@ -206,8 +206,8 @@ PSK privacy properties are orthogonal to security properties described in {{sec-
 Traditionally, TLS does little to keep PSK identity information private. For example,
 an adversary learns information about the external PSK or its identifier by virtue of it
 appearing in cleartext in a ClientHello. As a result, a passive adversary can link
-two or more connections together that use the same external PSK on the wire. Applications should
-take precautions when using external PSKs to mitigate these risks.
+two or more connections together that use the same external PSK on the wire. How best to
+mitigate these risks remains an open problem.
```
Comment (chris-wood) [inline draft-ietf-tls-external-psk-guidance.md]: How about:

```suggestion
two or more connections together that use the same external PSK on the wire. Techniques for mitigating
these risks require analysis and are out of scope for this document.
```

**Review by chris-wood — APPROVED** [review]

**Comment by chris-wood** [comment]
@cbartle891 please let me know what you think of the suggestion above!

**Comment by cbartle891** [comment]
@chris-wood I disagree that those analyses are out of scope since they concern the use of external PSKs, but if "out of scope" is understood to be synonymous with "we're not going to do it," then this change is fine. :)

**Comment by chris-wood** [comment]
>  but if "out of scope" is understood to be synonymous with "we're not going to do it," then this change is fine. :)

Indeed -- "out of scope for this document" means "we're not doing it in this document."

**Review by chris-wood — APPROVED** [review]

**Pull request merged**